modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -617,6 +617,21 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
     */
     CV_EXPORTS_W Net readNetFromDarknet(const String &cfgFile, const String &darknetModel = String());
 
+    /** @brief Reads a network model stored in <a href="https://pjreddie.com/darknet/">Darknet</a> model files.
+    *  @param cfgStream          stream to the .cfg file with text description of the network architecture.
+    *  @param darknetModelStream stream to the .weights file with learned network.
+    *  @returns Network object that ready to do forward, throw an exception in failure cases.
+    *  @returns Net object.
+    */
+    CV_EXPORTS_W Net readNetFromDarknet(std::istream &cfgStream, std::istream &darknetModelStream);
```

Comment:
Due `std::istream` has no wrapping policy we need to use `CV_EXPORTS` here.

Reply:
OK, now I will to replace `std::istream` to IStream of 3rdparty. Can I add `3rdparty/openexr/IlmImf/ImfIO.h` ? Or what do I should use?

Reply:
I got your saying. I will change std::istream to the other buffered structure like #9994.